apps/staging-config/package.json

```diff
@@ -22,7 +22,7 @@
     "jest": "27.4.5"
   },
   "scripts": {
-    "dev": "rm -f grouparoo_config.sqlite && cd node_modules/@grouparoo/core && DATABASE_URL=\"sqlite://grouparoo_config.sqlite\" NEXT_DEVELOPMENT_MODE=true GROUPAROO_RUN_MODE=\"cli:config\" WORKERS=0 ./bin/dev",
+    "dev": "cd node_modules/@grouparoo/core && NEXT_DEVELOPMENT_MODE=true GROUPAROO_RUN_MODE=\"cli:config\" WORKERS=0 ./bin/dev",
```

Comment:
I also updated the config and `.env.example` for staging-config to use in-memory SQLite, which 
1. is more like what customers will see
2. allows the application to hot-reload without issue when in development mode